### ISSUE TITLE
Chase Bank "Unrecognized Computer"

### ISSUE THREAD

**Issue by jonnorstrom** [searls/fine-ants#15]
Chase bank is intercepting the login attempt because it doesn't recognize my "computer" (therefore causing an error to be thrown and then the window is closed) which I'm assuming is due to some cookie stuff, something with the fact that a webdriver is doing this. Am I correct? Have you ran into this issue before? I'm forking to toy around with it.

**Comment by searls** [comment]
I haven't seen this but @McPolemic may have

**Comment by McPolemic** [comment]
Yep, you are absolutely correct. When the adapter was first written, Chase had a relatively normal login process. Now, they have three separate login forms that they cycle through to deter scraping and also check cookies as you're describing above.

Unfortunately, I haven't had time to look into it beyond that, but if you find anything I'd love improvements to this process 😄.

**Comment by jonnorstrom** [comment]
I forced the window to stay open, went through Chase's 2FA steps to learn about this "computer" and then upon later attempts, it works fine. Bit of a pain but it works. Two options I see are to craft a cookie to hand the original login attempt, or work out a "2FA" module to interact with that. My questions have been answered. Thanks for the quick feedback.
